.eslintignore

```diff
@@ -1,2 +1,3 @@
 typings/*.ts
 next-env.d.ts
+additional.d.ts
```

Comment:
What is this?

Reply:
In the Next.js documentation there is a section that describes how to add additional types (https://nextjs.org/docs/basic-features/typescript)

> Instead of editing next-env.d.ts, you can include additional types by adding a new file e.g. additional.d.ts and then referencing it in the [include](https://www.typescriptlang.org/tsconfig#include) array in your tsconfig.json.

I added this `additional.d.ts` file to reference `/// <reference types="next-react-svg" />` and added it to the `.eslintignore`. 

Do you think this is not a good way?

Reply:
I didn't know about that so I was curious is this something auto generated or so.
I'm fine with this, but usually on projects we put custom `.d.ts` files inside `typings` folder, except the `next-env.d.ts` which is auto generated.
You can see we already have `typings/*.ts` ignored

Reply:
Aha, I understand. Thx